src/game/camera.c

```diff
@@ -5212,7 +5332,11 @@ void set_camera_mode_8_directions(struct Camera *c) {
     if (c->mode != CAMERA_MODE_8_DIRECTIONS) {
         c->mode = CAMERA_MODE_8_DIRECTIONS;
         sStatusFlags &= ~CAM_FLAG_SMOOTH_MOVEMENT;
+#ifdef REONUCAM
+        s8DirModeBaseYaw = snap_to_45_degrees(s8DirModeBaseYaw);
```

Comment:
ahh i forgot about this.. we were once talking about having the camera use offset instead of baseyaw so that you could have the base yaw change dynamically.. maybe a problem for another day